### ISSUE TITLE
Card Component Styling Issue

### ISSUE THREAD

**Issue by vz-tech** [Comfortable-Coliving/ACCMobile#38]
Should be worked on after #19 is merged into main.

There are several styling issues regarding the use of the cards.
Please note that the four primary variant of cards are: event, eventMini, listing, listingV

Examining the iOS images (emulated using an iPhone 14 Pro Max) below, we have the following:
1) Adding too much text causes it to expand the title and force the eventMini card component to expand horizontally. This applies to the listMini card as well.
<img width="340" alt="image" src="https://github.com/Comfortable-Coliving/ACCMobile/assets/59082260/895ebe19-5da2-4603-98ac-b1298f3ba5e2">

2) As per Max's suggestion, more padding for bottom.
3) Images are not being properly contained in their view despite setting the width, height, maxWidth, maxHeight within the container they belong to. 
<img width="133" alt="image" src="https://github.com/Comfortable-Coliving/ACCMobile/assets/59082260/5f077264-97c5-49a2-ad6e-7407e50af00c">

4) Differences in how images are rendered inside a View in iOS and android. Please compare the foremost iOS images to the following ones in android (tested using an S20 FE with a 6.5 inch screen):

![image](https://github.com/Comfortable-Coliving/ACCMobile/assets/59082260/2ba10b4e-eb20-4cb4-bd63-59f169a51ae0)
![image](https://github.com/Comfortable-Coliving/ACCMobile/assets/59082260/df66c93a-7ba5-4d0f-b78b-3fb13db59b54)

As one can see, the image is not being contained properly in the View box it resides in.

I will look into resolving these issues in the future, but anybody interested in working on them can do so in main once it is merged, or immediately by switching into the current issue-19 branch. Thanks!
